setup.py

```diff
@@ -69,7 +69,8 @@ def is_ubuntu() -> bool:
     "pydocstyle",
     # Incompatible with current pylint-fixme-info==1.0.2
     # https://github.com/PyCQA/pylint/issues/5390
-    "pylint<2.12.0",
+    # "pylint<2.12.0",
```

Comment:
this change is not explained.

Reply:
I use pylint as part of my pre-push workflow (in `make tests`) and the newest version fails with extra warnings. The entry should eventually be restored to the commented one after those warnings are addressed (and the check can be added to the linters target in CI).

Reply:
Personally I would prefer to remove the line instead of commenting and a separate commit that we can later run `git revert` on

Reply:
Will do that.

Reply:
We can re-evaluate the need for pylint < 2.12 after addressing the issues, maybe pylint-fixme-info has already been updated to work with later versions.

Reply:
Will redo that in a separate commit and force-push to tidy things up.

Reply:
Ok, I won't pin pylint since I'm the only user. Once the warnings are addressed we can put it back in the linters target.